src/CodeGen_ARM.cpp

```diff
@@ -1204,10 +1205,11 @@ void CodeGen_ARM::visit(const Load *op) {
             Expr slice_ramp = Ramp::make(slice_base, ramp->stride, intrin_width);
             Value *ptr = codegen_buffer_pointer(op->name, op->type.element_of(), slice_base);
             ptr = builder->CreatePointerCast(ptr, i8->getPointerTo());
-            CallInst *call = builder->CreateCall(fn, vec(ptr, align));
+            Value *args[] = {ptr, align};
```

Comment:
Is this necessary? I'd be surprised if llvm::ArrayRef doesn't have an initializer list constructor...